java-runtimes-common/test-spring-application/README.md

```diff
@@ -0,0 +1,27 @@
+# Test Application
+
+This is a test application to be used in testing the Java Runtimes containers. It implements a set
+of http endpoints which exercise integrations with various Stackdriver APIs, in line with the 
+[Runtimes-common integration test framework](https://github.com/GoogleCloudPlatform/runtimes-common/tree/master/integration_tests#tests).
+
+The test application is a simple [Spring Boot](https://projects.spring.io/spring-boot/) application,
+which can be built in several different ways.
+
+## Supported profiles
+**Custom runtime profile** - stages the application for deployment to a custom runtime on App Engine Flexible:
```

Comment:
Can you please clarify what "stages" means, like that it prepares files in target/deploy ready to be used by "gcloud app deploy"?

Reply:
Yes, please.